.scripts/copy-shared-files.mjs

```diff
@@ -29,8 +30,11 @@ const POST_CREATE_EXCLUDE = [
   'signals-queries',
   'activities-cancellation-heartbeating',
   'replay-history',
+  'food-delivery'
 ];
 
+const NPMRC_EXCLUDE = [ 'food-delivery' ];
```

Comment:
Is this necessary?

Reply:
Yes, lockfile needs to be generated on install, else build fails.